public/pages/AnomalyCharts/utils/constants.ts

```diff
@@ -88,3 +88,12 @@ export enum LIVE_CHART_CONFIG {
   REFRESH_INTERVAL_IN_SECONDS = 30 * 1000, //poll anomaly result every 30 seconds
   MONITORING_INTERVALS = 60,
 }
+
+export const DEFAULT_ANOMALY_SUMMARY = {
+  anomalyOccurrence: 0,
+  minAnomalyGrade: 0,
+  maxAnomalyGrade: 0,
+  minConfidence: 0,
+  maxConfidence: 0,
```

Comment:
Small: for these 4 fields (min/max grade/confidence), can we change these to be strings as well, and display something like '-' here instead? Since these fields are irrelevant in this case, it seems more appropriate than just showing 0s.

Reply:
This is reviewed by UX designer. Will keep this as is currently.

Reply:
Got it, sounds good.